src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -57,7 +60,12 @@ public static UserStore getInstance() {
    * @param persistentStorageAgent a mock used for testing
    */
   public static UserStore getTestInstance(PersistentStorageAgent persistentStorageAgent) {
-    return instance = new UserStore(persistentStorageAgent);
+    instance = new UserStore(persistentStorageAgent);
+    instance.setActivityStore(ActivityStore.getTestInstance(persistentStorageAgent));
+
+    // hard-coded initial Admin:
+    instance.addUser("Admin01", "AdminPass01", /* admin= */ true);
```

Comment:
Why do we need the admin when running tests? I was thinking that it was only necessary for the non-test part.

Reply:
There are actually tests for the initial admin to check if they are added when creating a new UserStore

Reply:
<img width="890" alt="screen shot 2018-06-29 at 1 40 37 pm" src="https://user-images.githubusercontent.com/31927226/42106448-1ece313e-7ba2-11e8-8d1e-5860ded693a6.png">
I don't think it is necessary to add that into getTestInstance(). I just removed it and this is the only error I got. It is a normal error.

Reply:
Yeah, so it's testing if the hard-coded admin is added. It's expecting 3(1 initial admin + 2 later added) and it's getting 2(no initial admin + 2 later added). I feel like the point of testing is to see if it's behaving as expected (we are expecting 1 initial admin), so I kept the test instance matching with the real instance. 
BTW, if you change that 3 to a 2 in the testing, you will get an assert error later (that was the error I was getting last night)
What do you think?

Reply:
The error is coming from the boolean check you added inside the for loop to check for admin. But I think you are right about testing that it is actually creating another admin in the back. Yeah, we should keep it then as long as everything else is working